creditcoin-js/package.json

```diff
@@ -9,15 +9,15 @@
     ],
     "scripts": {
         "build": "tsc",
-        "example": "ts-node ./src/examples/loan-cycle.ts",
+        "example": "ts-node --files ./src/examples/loan-cycle.ts",
```

Comment:
yeah, it loads these lines in the tsconfig.json

https://github.com/gluwa/creditcoin/blob/3815fd22f1a4b08cffb3a34a6dd47812f628a3d4/creditcoin-js/tsconfig.json#L19-L20

according to the  ` ts-node --help`

```
 --files                         Load `files`, `include` and `exclude` from `tsconfig.json` on startup
```

I was getting a bunch of compilation errors otherwise.